examples/palettization/dkm_palettization.py

```diff
@@ -0,0 +1,225 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
https://github.pie.apple.com/mlpt/phoenix/tree/develop/test/tutorials
Are we testing this file somewhere else?